README.md

```diff
@@ -18,10 +18,9 @@ Project is written in TypeScript and provides type safety out of the box. No Flo
     -   [`<Observer/>`](#observer)
     -   [`observer<P>(baseComponent: FunctionComponent<P>, options?: IObserverOptions): FunctionComponent<P>`](#observerpbasecomponent-functioncomponentp-options-iobserveroptions-functioncomponentp)
     -   [`useObserver<T>(fn: () => T, baseComponentName = "observed", options?: IUseObserverOptions): T`](#useobservertfn---t-basecomponentname--%22observed%22-options-iuseobserveroptions-t)
-    -   [`useObservable<T>(initialValue: T): T`](#useobservabletinitialvalue-t-t)
```

Comment:
I am thinking we shouldn't be probably removing those hooks in this PR. They should become deprecated and released in 1.X with those new hooks. Then the 2.X can come out with those removed. 

Personally, I don't want to be refactoring whole apps right away :)

Reply:
Ok, reverted, added warning instead